examples/integration_test.py

```diff
@@ -265,6 +266,8 @@ def verify_avro():
         dict(value=float_value, value_schema=prim_float, key=str_value, key_schema=prim_string),
         dict(value=str_value, value_schema=prim_string, key={'name': 'abc'}, key_schema=basic),
         dict(value=str_value, value_schema=prim_string, key=float_value, key_schema=prim_float),
+        dict(value='', value_schema=prim_string, key=0., key_schema=prim_float),
+        dict(value=0., value_schema=prim_float, key='', key_schema=prim_string),
```

Comment:
It is too bad the consumer doesn't seem to actually verify the consumed messages, but we'll leave that for later.

Reply:
do you want to verify all keys and values or just the falsy ones

Reply:
signed, sealed, delivered. I moved forward with only verifying the `Falsy` values on the consumer end. If this is not sufficient let me know and I'll act accordingly